install_template/templates/products/migration-toolkit/index.njk

```diff
@@ -24,14 +24,14 @@ legacyRedirects:
 {% endblock frontmatter %}
 {% block navigation %}
 {{- super() -}}
-- install_on_mac
-- install_on_windows
+- macos
```

Comment:
We are no longer building macOS installers. References to macOS should be removed.